samples/boards/stm32/spi_flash/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _spi-nor-sample:
+
+FLASH NOR Sample in MemoryMapped
+################################
+
+Overview
+********
+
+This sample demonstrates using the flash API on a SPI NOR serial flash
+memory device.  While trivial it is an example of direct access to the
+external NOR quad or octo flash in MemoryMapped mode.
+The Nor quad/octo Flash is configured in MemoryMapped mode at the device init.
+Then read/arite access is done with memcopy.
+The sample is writing few bytes and read them back at offset 0
+Then read bytes at offset=SPI_FLASH_SECTOR_SIZE
```

Comment:
This kind of application is useful for our own needs, but not much for users.
Do we need to have it in tree ?

An MEMMAP XIP  basic hello world sample would be great though

Reply:
A "MEMMAP xip application" is more complex to handle with --sysbuild building and linking, with memory partitioning and flashing to internal and external memories.
That sample is light : read few bytes memory mapped